### PR TITLE
#116592613 Display Error when one tries to send invite on an empty username

### PR DESCRIPTION
**Pull request by vickris**
#### What does this PR do?
- Fixes the issue where one could send an invite without passing in a username
#### Description of Task to be completed?
- Check if username is empty when making the post request
- If username is empty display an error using sweet alert that the username cannot be empty
#### How should this be manually tested?
- Login into the admin dashboard (for those with admin privileges)
- Try sending an invite without passing the username
- An error prompt will appear telling you that the username cannot be empty
#### What are the relevant pivotal tracker stories?
#116592613
#### Screenshots (if appropriate)

screen shot 2016-03-30 at 12 49 17 pm

### CODE REVIEW

**Comment by andela-oadebayo** [comment]
**Code Functionality - Nit Pick**
Fix cases of empty spaces.

**Comment by andela-oadebayo** [comment]
**Code Functionality Nit Pick**
@andela-cvundi  Good Job.

File: public/js/admin.js [hunk 1]
```diff
@@ -128,6 +128,20 @@ $(document).ready(function(){
         var url       = "/dashboard/user/create";
         var token     = $("#_token").val();
         var username  = $("#username").val();
+
+        //throw error if nothing was passes in the username field
+        username = username.trim();
```
Comment (unicodeveloper) [inline public/js/admin.js]: **Code Quality - Nitpick**

Avoid duplication. The trim method could have been applied on line 130 like so:

``` js
var username = $("#username").val().trim();
```

**Comment by vickris** [comment]
@unicodeveloper You can merge the PR. Took care of the whitespace issue

**Pull request merged**